beta-src/src/index.tsx

```diff
@@ -1,16 +1,20 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import { CssBaseline, ThemeProvider } from "@mui/material";
+import { Provider } from "react-redux";
 import webDiplomacyTheme from "./webDiplomacyTheme";
 import "./assets/css/index.css";
 import App from "./App";
+import { store } from "./state/store";
```

Comment:
@nebevets What are your thoughts on us implementing this? https://github.com/trivago/prettier-plugin-sort-imports 

There are several similar solutions, I think they just vary slightly.  I think the many differences are how they are grouped and/or removing unused imports.  

Lmk your thoughts.

Reply:
it looks cool, but is there a prettier config for this? my other concern is that eslint and/or typescript i have noticed throws errors based on the ordering of imports...i wonder will this play nicely with that? maybe we could make this an infra task...time box it to 4 hours and make a pr to get something into the project that works nicely to order our imports better.